src/tables/Project.sol

```diff
@@ -30,14 +33,22 @@ contract Project is TablelandController, Ownable {
                 "proposalLink text,"
                 "finalReportIPFS text,"
                 "finalReportLink text,"
-                "contributors text",
+                "contributors text,"
```

Comment:
is this a list of contributors? I guess now this will be covered with the hats for the team